read.pl

```diff
@@ -128,16 +128,18 @@ sub inner_read_report {
     if ( $_ =~ /^\s*(\d+) / ) {
       $report2taskid[$i] = $1;
       $taskid2report[$1] = $i;
+      audit("inner_read_report: report2taskid[$i]=$1, taskid2report[$1]=$i")
     } else {
       $report2taskid[$i] = $prev_id;
-      $taskid2report[$i] = $prev_id;
+      audit("inner_read_report: report2taskid[$i]=$prev_id")
     }
     $prev_id = $report2taskid[$i];
     $i++;
   }
   close(IN);
 
   if ( $#report_tokens > -1 ) {
+    # WHY?!!!!
```

Comment:
I have the same comment for a lot of VIT's code. Is your comment about why the if statement conditions on that particular condition, or is it about the line below the comment? For comments like these, let's use something more standard, like:

`# FIXME: I don't see why [insert reason] Either comment why or try removing? bz`

You do not need to sign (as I put above with bz), but I personally find it helpful.